airflow-comparison.md

```diff
@@ -0,0 +1,47 @@
+## Lineage
+
+Data lineage broadly refers to a collection of tools, practices and approaches which allow the tracking of the flow of data within a data pipeline. Data lineage tools maintain a record of data within its lifecycle, including its source and any transformations applied to it. The main purpose of data lineage is to provide a platform for ensuring data quality, as well as a way of troubleshooting any errors which arise within the data.
+
+OpenLineage is 'an open platform for collection and analysis of data lineage’. It is the defacto industry standard for lineage collection, and it will be the basis for our comparison of lineage capabilities between Airflow and VDK.
+
+Apache Airflow offers support for OpenLineage compliant lineage data collection through the `airflow-openlineage` package; however, out-of-the-box support is only offered for PostgreSQL, Snowflake, MySQL, BigQuery, and the Great Expectations data quality validation framework. Users are expected to implement their own solution or borrow an existing one should they want to collect lineage for queries ran against a different database.
```

Comment:
This is the catchy part for me, the actual comparison (purpose) of the article. Maybe start the sentence with the verb "Comparing ..., ..." or similar. Also, comparison usually includes a table-like structure, where you cover different aspects / points you test. It's easier to compare one next to the other.